gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/rest/goldens/HttpJsonComplianceStub.golden

```diff
@@ -350,6 +351,9 @@ public class HttpJsonComplianceStub extends ComplianceStub {
                   .build())
           .build();
 
+  private static final Map<String, String> operationCustomHttpBindings =
```

Comment:
We probably don't want to create unused variables if we can avoid it in the generator.